backend/src/bundles/talent-badges/talent-badge.service.ts

```diff
@@ -0,0 +1,56 @@
+import { ErrorMessages } from '~/common/enums/enums.js';
+import { type Service } from '~/common/types/types.js';
+
+import { type TalentBadgeEntity } from './talent-badge.entity.js';
+import { type TalentBadgeRepository } from './talent-badge.repository.js';
+import {
+    type TalentBadgeCreate,
+    type TalentBadgeUpdate,
+} from './types/types.js';
+
+class TalentBadgeService implements Service {
+    private talentBadgeRepository: TalentBadgeRepository;
+
+    public constructor(talentBadgeRepository: TalentBadgeRepository) {
+        this.talentBadgeRepository = talentBadgeRepository;
+    }
+
+    public findAll(): Promise<{ items: TalentBadgeRepository[] }> {
+        throw new Error(ErrorMessages.NOT_IMPLEMENTED);
+    }
+
+    public find(): Promise<TalentBadgeRepository> {
+        throw new Error(ErrorMessages.NOT_IMPLEMENTED);
+    }
+
+    public async create(badge: TalentBadgeCreate): Promise<TalentBadgeEntity> {
+        return await this.talentBadgeRepository.create(badge);
```

Comment:
I see a lot same return statements with await.
You have already return Promise, you do not need to await it
Read more [no-return-await](https://kanby.medium.com/%D0%BF%D0%BE%D1%87%D0%B5%D0%BC%D1%83-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D1%8C-return-await-%D0%BF%D0%BB%D0%BE%D1%85%D0%B0%D1%8F-%D0%B8%D0%B4%D0%B5%D1%8F-e87b70015f0c)
```suggestion
        return this.talentBadgeRepository.create(badge);
```

Reply:
thanks